helpers/createZipFile.js

```diff
@@ -0,0 +1,38 @@
+const glob = require('glob');
+const JSZip = require('jszip');
```

Comment:
This variable name violates the convention :)

Reply:
Which one do you think it should be: `jsZip` or `JsZip`? (this is a constructor)

Reply:
Yeah, I think `JsZip` is good